CorsixTH/Src/th_gfx_font.h

```diff
@@ -206,80 +206,80 @@ class THFreeTypeFont final : public THFont
 
         @param pBitmapFontSpriteSheet The sprite sheet of the bitmap font.
     */
-    FT_Error matchBitmapFont(sprite_sheet* pBitmapFontSpriteSheet);
+    FT_Error match_bitmap_font(sprite_sheet* pBitmapFontSpriteSheet);
 
     //! Set the ideal character size using pixel values.
     /*!
         Note that the given size might be changed a small amount if doing so
         would result in a much nicer rendered font. This must be called after
         setFace().
     */
-    FT_Error setPixelSize(int iWidth, int iHeight);
+    FT_Error set_ideal_character_size(int iWidth, int iHeight);
 
-    THFontDrawArea getTextSize(const char* sMessage, size_t iMessageLength,
+    text_draw_dimensions get_text_dimensions(const char* sMessage, size_t iMessageLength,
                                int iMaxWidth = INT_MAX) const override;
 
-    void drawText(render_target* pCanvas, const char* sMessage,
+    void draw_text(render_target* pCanvas, const char* sMessage,
                   size_t iMessageLength, int iX, int iY) const override;
 
-    THFontDrawArea drawTextWrapped(render_target* pCanvas, const char* sMessage,
+    text_draw_dimensions draw_text_wrapped(render_target* pCanvas, const char* sMessage,
             size_t iMessageLength, int iX, int iY,
             int iWidth, int iMaxRows = INT_MAX, int iSkipRows = 0,
-            eTHAlign eAlign = eTHAlign::left) const override;
+            align eAlign = align::left) const override;
 
 private:
-    struct cached_text_t
+    struct cached_text
     {
         //! The text being converted to pixels
-        char* sMessage;
+        char* message;
 
         //! Raw pixel data in row major 8-bit greyscale
-        uint8_t* pData;
+        uint8_t* data;
 
         //! Generated texture ready to be rendered
-        SDL_Texture* pTexture;
+        SDL_Texture* texture;
 
         //! The length of sMessage
-        size_t iMessageLength;
+        size_t message_length;
 
         //! The size of the buffer allocated to store sMessage
-        size_t iMessageBufferLength;
+        size_t message_buffer_length;
 
         //! Width of the image to draw
-        int iWidth;
+        int width;
 
         //! Height of the image to draw
-        int iHeight;
+        int height;
 
         //! The width of the longest line of text in in the textbox in pixels
-        int iWidestLine;
+        int widest_line_width;
 
         //! X Coordinate trailing the last character in canvas coordinates
-        int iLastX;
+        int last_x;
 
         //! Number of rows required
-        int iNumRows;
+        int row_count;
 
         //! Alignment of the message in the box
-        eTHAlign eAlign;
+        align alignment;
 
-        //! True when the pData reflects the sMessage given the size constraints
-        bool bIsValid;
+        //! True when the data reflects the message given the size constraints
```

Comment:
Such a shame that we loose the link to the variables by the rename operation, doxygen would need a `#` prefix (or a full `classname::` prefix) to preserve the link.

Reply:
Ok, I didn't know it generated the link before.